src/app/[...slug]/page.client.tsx

```
@@ -71,18 +49,22 @@ const SlugClientPage = ({ initialData }: SlugClientPageParams) => {
-      // 각 이미지 다운로드
-      if (frontFileName) {
-        handleSaveImg(frontImgData, frontFileName);
-      }
-      if (backFileName) {
-        handleSaveImg(backImgData, backFileName);
+
+      if (flipCardRef.current) {
+        const { front, back } = await flipCardRef.current.exportCardImages();
+
+        const savePromises: void[] = [];
+        if (front) {
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)